internal/impl/kafka/enterprise/redpanda_migrator_bundle_input.tmpl.yaml

```diff
@@ -137,9 +137,9 @@ tests:
               consumer_group: "migrator"
             processors:
               - mapping: meta input_label = "redpanda_migrator"
-          - kafka_franz:
+          - redpanda_migrator_offsets:
               seed_brokers: [ "127.0.0.1:9092" ]
-              topics: [ "__consumer_offsets" ]
+              topics: [ "foobar" ]
```

Comment:
Why is this `foobar`?

Reply:
But we don't want a filter by default? Or this should be configurable? I don't understand why this is hardcoded the way it is.

Reply:
Answered on Slack. TL; DR: This is the test section of the template where I'm asserting that the topic filter for `redpanda_migrator_offsets` is set to a specific topic. We don't want to send consumer group updates for all topics to the `redpanda_migrator_offsets` output.